templates/show-task-table.html

```diff
@@ -33,13 +33,33 @@
                 <tbody>
                 {% for task in tasks %}
                   <tr>
-                    <td><a href="https://en.wikipedia.org/wiki/{{ task.article_name|replace:" " }}">{{ task.article_name }}</a>
-                         {% if logged_in_user and task.status == "open" or logged_in_user and task.status == "claimed" and task.claimed_by == logged_in_user %} <i class="fa fa-pencil-square-o" onclick="taskForm({{forloop.counter0}})"></i> {% endif %}
-                    </td>
+                    <td><a href="https://en.wikipedia.org/wiki/{{ task.article_name|replace:' ' }}">{{ task.article_name }}</a></td>
```

Comment:
Hm, sorry, I forgot how this works -- is `replace` a Django filter? I don't see it in the [documentation](https://docs.djangoproject.com/en/2.1/ref/templates/builtins/).

And what is this replacing? Spaces for ...?

Reply:
Oh, I see, I was looking for it on the master branch but it's not there yet. We should probably talk about making pull requests tidier on our next sync, it gets a confusing to review things when there's more than one development branch and the pull requests contain big commits just resolving merge conflicts between two different branches. git can be hard to tame, but I can try to help if it's getting in the way of your workflow.

Reply:
Once the PR is approved I'll cherry-pick the commits and merge into master.